### PR TITLE
chore(cmd/gossamer): Request chain spec as an argument in `import-runtime` command

### LINKED ISSUE #3013: remove the call to `GetGssmrGenesisPath` from `createGenesisWithRuntime`
Currently, the function `createGenesisWithRuntime` (`cmd/gossame/import_runtime`) calls the `utils.GetGssmrGenesisPath()`.

In order to address the epic https://github.com/ChainSafe/gossamer/issues/2895 we should update this function to don't call the `GetGssmrGenesisPath` function

### PR DESCRIPTION
**Pull request by EclesioMeloJunior**
## Changes

- `import-runtime` argument was tightly coupled with the `gssmr` genesis spec file so in order to remove this dependency I've introduced the genesis spec file as an argument to the `import-runtime` command

## Tests

```sh
go test -tags integration github.com/ChainSafe/gossamer/cmd/gossamer
```

## Issues

- Closes #3013

## Primary Reviewer

@jimjbrettj

### CODE REVIEW

File: lib/utils/utils.go [hunk 1]
```diff
@@ -148,16 +148,6 @@ func KeystoreFilepaths(basepath string) ([]string, error) {
 	return keys, nil
 }
 
-// GetGssmrGenesisRawPath gets the gssmr raw genesis path
-// and returns an error if it cannot find it.
-func GetGssmrGenesisRawPath() (path string, err error) {
```
Comment (jimjbrettj) [inline lib/utils/utils.go:153]: Love to see it!

**Review by jimjbrettj — APPROVED** [review]
Love to see it!

File: cmd/gossamer/main.go [hunk 2]
```diff
@@ -186,11 +186,13 @@ func importStateAction(ctx *cli.Context) error {
 func importRuntimeAction(ctx *cli.Context) error {
 	arguments := ctx.Args()
 	if len(arguments) == 0 {
-		return fmt.Errorf("no args provided, please provide wasm file")
+		return fmt.Errorf("no args provided, please provide wasm file and the genesis spec file")
 	}
 
 	fp := arguments[0]
-	out, err := createGenesisWithRuntime(fp)
+	genesisChainSpec := arguments[1]
```
Comment (qdm12) [inline cmd/gossamer/main.go:193]: ⚠️ check the length of arguments is at least 2

**Review by qdm12 — APPROVED** [review]
Just a minor thing to change 😉

**Pull request merged**